test/evergreen/find_cmake.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Adapted 'find_cmake' from mongo-c-driver evergreen infrastructure:
+#   https://github.com/mongodb/mongo-c-driver/blob/master/.evergreen/find-cmake.sh
+find_cmake ()
```

Comment:
There seems to be some indentation inconsistency in this function.

Reply:
Fixed the indentation :)